src/main/java/Main.java

```diff
@@ -1,6 +1,141 @@
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        setPeoples();
+        setGoods();
+        setRetry();
     }
-}
\ No newline at end of file
+
+    public static void setPeoples() {
+        System.out.println(divider);
+        System.out.println(divider);
+
+        System.out.printf("%sНа сколько человек поделить счёт?\n", Colors.CYAN);
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+
+            if (scanner.hasNextInt()) {
+                int count = scanner.nextInt();
+
+                if (count <= 1) {
+                    System.out.printf("%sКоличество человек должно быть больше %s1.%s\n", Colors.CYAN, Colors.RED_BOLD, Colors.RESET);
+                    continue;
+                }
+
+                peoples = count;
+                break;
+            }
+
+            System.out.printf("%sЧисло человек введено не корректо.%s\n", Colors.RED_BOLD, Colors.RESET);
+        }
+
+        System.out.println(divider);
+        System.out.println(divider);
+    }
+
+    public static void setGoods() {
+        System.out.printf("%sДобавление товара в корзину.\n", Colors.CYAN);
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+
+            String productName = setProductName();
+            float productPrice = setProductPrice();
+
+            if (isExit) {
+                isExit = false;
+                break;
+            } else {
+                goods.add(new Product(productName, productPrice));
+                System.out.printf("\n%sДобавим еще товар?\n", Colors.CYAN);
+            }
+        }
+
+        System.out.println(divider);
+        System.out.println(divider);
+
+        goods.show(peoples);
+    }
+
+    private static String setProductName() {
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+
+            System.out.printf("%sВведите название товара." + exitStr, Colors.CYAN);
+
+            if (scanner.hasNextInt()) {
+                System.out.printf("%sНазвание товара введено не корректо.%s\n", Colors.RED_BOLD, Colors.RESET);
+                continue;
+            }
+
+            String name = scanner.nextLine();
+
+            exit(name);
+
+            return isExit ? "" : name;
+        }
+
+    }
+
+    private static float setProductPrice() {
+        if (isExit) return 0.0f;
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+
+            System.out.printf("%sВведите стоимость товара." + exitStr, Colors.CYAN);
+
+            if (!scanner.hasNextFloat()) {
+
+                exit(scanner.nextLine());
+
+                if (isExit) return 0.0f;
+
+                System.out.printf("%sСтоимость товара введена не корректо.%s\n", Colors.RED_BOLD, Colors.RESET);
+                continue;
+            }
+
+            float price = scanner.nextFloat();
+
+            if(price < 0) {
```

Comment:
```suggestion
            if (price < 0) {
```
Перед коммитом стоит приводить файлы к правильному стилю, в студии это легко делать путём вызова горячей комбинации клавиш (она зависит от ОС), её можно посмотреть в меню Code → Reformat Code

Reply:
Исправил